src/select-list-view.ts

```diff
@@ -32,10 +55,12 @@ module.exports = class SelectListView {
     const didLoseFocus = this.didLoseFocus.bind(this)
     editorElement.addEventListener('blur', didLoseFocus)
 
-    // When clicking the scrollbar of the items list, a blur event will be triggered
-    // on the query editor element, but we don't want to treat that as a cancellation.
-    // This mousedown listener allows us to detect this case and restore focus to the
-    // query editor. This is based on https://stackoverflow.com/a/1480178.
+    /**
+     * When clicking the scrollbar of the items list, a blur event will be triggered
+     * on the query editor element, but we don't want to treat that as a cancellation.
+     * This mousedown listener allows us to detect this case and restore focus to the
+     * query editor. This is based on https://stackoverflow.com/a/1480178.
+     */
```

Comment:
Can you change this back to the previous line-style comment?

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/e5df058b6b014c6708eb6cb173ff25c8ef0cff4e